lib/pinpoint_supported_countries.rb

```diff
@@ -145,8 +147,8 @@ def name_to_iso_code(name)
     }[name]
   end
 
-  def trim_digits_spaces(str)
-    str.gsub(/\s{2,}/, ' ').gsub(/[\d\s]+$/, '')
+  def trim_spaces(str)
+    str.gsub(/\s{2,}/, ' ').gsub(/[\s]+$/, '')
```

Comment:
Minor: No need for character class anymore if we're only looking for whitespace now

```suggestion
    str.gsub(/\s{2,}/, ' ').gsub(/\s+$/, '')
```

Reply:
good catch, thanks